erpnext/docs/user/manual/en/schools/setup/room.md

```diff
@@ -1,6 +1,19 @@
 # Room
 
+A classroom is a learning space, a room in which both children and adults learn about things. Classrooms are found in educational institutions of all kinds, from preschools to universities, and may also be found in other places where education or training is provided, such as corporations and religious and humanitarian organizations. The classroom attempts to provide a space where learning can take place uninterrupted by outside distractions.
+
```

Comment:
such definitions are not required. Most people know what a class room is. Also if you are copying from somewhere. Please attribute!

What users need to know is where a **Classroom** record is linked.